CHANGELOG.md

```diff
@@ -8,32 +8,86 @@ core section.***
 - ***When possible please provide the number of the pull request(s) containing the 
 changes in the following format: PR #1234***
 
-### LORIS 23.0 (Released: ??)
+## LORIS 23.0
 
+### Core
+#### Features
+- Add configuration option to specify the format in which dates are displayed. (PR #5004)
+- Configurable upload directory for data_release and document_repository modules (PR #5815)
+- New radio components for forms. (PR #5846)
 
-#### Core
+#### Updates and Improvements
+- PHP minimum requirements raised to PHP 7.3 (PR #5723)
 - Menus are now maintained by modules and no longer in the SQL database (PR #5839)
-- Very old instruments relying on QuickForm may have issues due to code changes (PR #4928)
-- Unix user permissions have been updated which may affect access to files. New
-documentation for file permissions has been added to the README.md file (PR #5323)
+- New script created for streamlining instantiation of the Raisinbread demonstration 
+database (PR #5260)
+- New documentation for file permissions has been added to the README.md file. (PR #5323)
+- Study progression performance improvement (PR #5887)
 
-#### Modules 
+#### Bug Fixes
+- Fix width of select elements that have no label (PR #5932)
```

Comment:
```suggestion
```

I don't think we need to mention this. It was a bug fix in an existing component for a new module that I was working on, no other places in LORIS seem to use/depend on it.